### DISCUSSION TITLE
How to include JavaFX in micro soft agent ubuntu 18.4 jdk 8 Open

### DISCUSSION THREAD

**Discussion by singarayanGitHub** [actions/virtual-environments#3801] [Installed software]
The build is failing in azure pipeline as javafx is missing in the open jdk. How can we fix this issue? we are using oracle jdk 8 in our local which is working fine.

**Comment by miketimofeev** [comment]
Hi @singarayanGitHub! What OS do you use and what is the use case?

**Comment by singarayanGitHub** [comment]
Thanks for your reply

I am using Ubuntu 18.4 in Azure Microsoft agent. The build is triggered in
azure devops build pipeline where open jdk 8 used.

Thanks

**Reply by miketimofeev** [reply]
It looks like javafx was unbundled from openjdk package:
https://adoptopenjdk.net/faq.html#openjfxfaq
There are a few options in the article how to install openjfx
